src/library/lib.js

```diff
@@ -148,6 +148,7 @@ filepicker.extend(function(){
         options.storeLocation = store_options.location || 'S3';
         options.storePath = store_options.path;
         options.storeContainer = store_options.storeContainer || store_options.container;
+        options.storeRegion = store_options.storeRegion || store_options.region;
```

Comment:
We don't need support both key names here ```store_options.storeRegion || store_options.region;```.
I guess it should be only ```store_options.region```.

We support ```store_options.storeContainer || store_options.container``` because of some legacy implementations.